python/src/server.py

```diff
@@ -63,7 +68,8 @@ def get_verification_info_by_token(public_token: str):
         data using a front end public_token """
 
     # First exchange public_token to access_token
-    access_token = api_client.get_access_token(public_token)
```

Comment:
please test all products

Reply:
all products have already been corrected and tested for the change to get_access_token